### ISSUE TITLE
manual/scalaGuide/main/http/ScalaBodyParsers.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#1007]
- https://github.com/playframework-ja/translation-project/tree/2.3.x/manual/scalaGuide/main/http/ScalaBodyParsers.md
- diffs between [2.2.0/manual/scalaGuide/main/http/ScalaBodyParsers.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/scalaGuide/main/http/ScalaBodyParsers.md) and [2.3.x/manual/scalaGuide/main/http/ScalaBodyParsers.md](https://github.com/playframework-ja/translation-project/tree/2.3.x/manual/scalaGuide/main/http/ScalaBodyParsers.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/main/http/ScalaBodyParsers.md    2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/scalaGuide/main/http/ScalaBodyParsers.md   2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Body parsers

 ## What is a body parser?
@@ -89,14 +90,15 @@

 Text based body parsers (such as **text**, **json**, **xml** or **formUrlEncoded**) use a maximum content length because they have to load all of the content into memory. 

-There is a default content length (the default is 100KB), but you can also specify it inline:
+There is a default maximum content length (the default is 100KB), but you can also specify it inline:

 @[body-parser-limit-text](code/ScalaBodyParser.scala)

-
 > **Tip:** The default content size can be defined in `application.conf`:
 > 
 > `parsers.text.maxLength=128K`
+> 
+> Unit sizes are defined in **Size in bytes format** section of the [[Configuration]] page.

 You can also wrap any body parser with `maxLength`:


```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1084
